storybook/web/config.js

```diff
@@ -4,6 +4,7 @@ import { addDecorator, configure } from '@storybook/react';
 import { setOptions } from '@storybook/addon-options';
 import { withKnobs } from '@storybook/addon-knobs';
 import theme, { injectBaseStyles } from '../../src/theme';
+import layoutDecorator from '../decorators/layout';
```

Comment:
How about instead of a custom implementation, we use something that already exists? Something like [@storybook/addon-centered](https://github.com/storybooks/storybook/tree/master/addons/centered) ?

Reply:
It doesn't add any spacing on the sides.

addon-centered:
<img width="1299" alt="screen shot 2018-05-04 at 7 28 19 am" src="https://user-images.githubusercontent.com/15325890/39610280-ed882762-4f6c-11e8-83c5-a3f645de1446.png">

this PR:
<img width="1299" alt="screen shot 2018-05-04 at 7 29 20 am" src="https://user-images.githubusercontent.com/15325890/39610288-f431acfa-4f6c-11e8-9dbe-440d8bc19ddb.png">

I can change the PR to use addon-centered if you feel that would be better though.

Reply:
Hey @tallpants, apologies on the late reply was a long weekend. 
Let's stick with addon-centered, since it gives a more realistic representation when using 100% width components. If we see a stronger use case for a custom implementation we can revisit then. Let me know what you think.

Reply:
@lakshyaranganath done!